packages/pwa-kit-create-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pwa-kit-create-app",
-  "version": "1.4.0",
+  "version": "1.5.0-dev",
```

Comment:
For some reason (maybe from merge conflict?), the generator's version number was not bumped up, while all other packages were. This made my testing changes tricky. For like 1-2 hours, I couldn't figure out why I wasn't able to see my changes. It turns out that when running our create-mobify-app-dev script, I was testing against the older, publicly available version.

In this PR, I added a check to make sure that all package version numbers are the same.